src/widgets/scale/lv_scale.c

```diff
@@ -636,11 +636,23 @@ static void scale_draw_label(lv_obj_t * obj, lv_event_t * event, lv_draw_label_d
         label_dsc->text_local = 1;
     }
 
+    int32_t translate_x = lv_obj_get_style_translate_x(obj, LV_PART_INDICATOR);
+    int32_t translate_y = lv_obj_get_style_translate_y(obj, LV_PART_INDICATOR);
+    int32_t label_rotation = lv_obj_get_style_transform_rotation(obj, LV_PART_INDICATOR);
+    int32_t translate_rotation = 0;
+
     if((LV_SCALE_MODE_VERTICAL_LEFT == scale->mode || LV_SCALE_MODE_VERTICAL_RIGHT == scale->mode)
        || (LV_SCALE_MODE_HORIZONTAL_BOTTOM == scale->mode || LV_SCALE_MODE_HORIZONTAL_TOP == scale->mode)) {
-        scale_get_label_coords(obj, label_dsc, tick_point_b, &label_coords);
+        lv_point_t label_origin;
+        label_origin.x = tick_point_b->x + translate_x;
+        label_origin.y = tick_point_b->y + translate_y;
+        scale_get_label_coords(obj, label_dsc, &label_origin, &label_coords);
+        label_rotation = (label_rotation & LV_SCALE_ROTATION_ANGLE_MASK) * 10U;
```

Comment:
Please do not multiple by 10 to allow setting the angle with 0.1 degree precision and for consistency with other angle units.